src/user.js

```diff
@@ -33,12 +48,30 @@ class User {
     });
   }
 
+  addToCook(recipe) {
+    if (typeof recipe === 'object') {
+      this.recipesToCook.push(recipe)
+    }
+  }
+
+  removeFromRecipesToCook(recipe) {
+    let cooked = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(cooked, 1)
+  }
+
+  filterRecipesToCook(tag) {
+    return this.recipesToCook.filter(recipe => {
+      return recipe.tags.includes(tag);
+    })
+  }
+
+
```

Comment:
Looks good to me, need to consider dom functionality for this. Probably something similar to the way the displaying of recipe directions is handled could work.